index.js

```diff
@@ -112,6 +122,43 @@ class Conf {
 		}
 	}
 
+	_migrate(migrations = {}, versionToMigrate = '0.0.0') {
```

Comment:
I don't see the value of having default parameters on this function.

Reply:
The first one I could agree with you since it's only possible to arrive at this code block with a positive value in the migrations object.

Do you think the last one should be removed as well?  It's possible for the options.projectVersion to be undefined, which would cause this code to break.
One could also argue that if the migrations object is specified, but the projectVersion is not, we should throw an error.

Thoughts on this?

Reply:
Agree!

Reply:
Done, please review again 😅